usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -41,6 +43,18 @@ def test_sles_hardened(host, get_release_value, is_sles_sap, is_sle_micro):
     if is_sle_micro():
         pytest.skip('Skipping SLE Micro')
 
+    scap_report = os.environ.get("SCAP_REPORT", "")
+    if scap_report:
+        if not scap_report.endswith(".html"):
+            print("Ignoring SCAP_REPORT: {}".format(scap_report))
+        elif host.file(scap_report).exists:
+            # Do not retry this expensive test if the report was generated
+            with open(LOG_FILE) as file:
+                reason = file.read()
+            pytest.fail(reason=reason, pytrace=False)
```

Comment:
> Why are we failing here? What if the test ran and it passed? Now the test gets marked as fail?

We're failing here because we can safely assume that if the SCAP report exists it's because `oscap` failed, otherwise we wouldn't be here.  And the log file is only written in case of failure.

Perhaps I should add a comment?

Reply:
Added a comment and dropped logfile.